src/amp/components/Ad.test.tsx

```diff
@@ -47,28 +49,34 @@ describe('AdComponent', () => {
 		expect(ampAdElement).not.toBeNull();
 
 		if (ampAdElement) {
-			const usRtcAttribute = ampAdElement[0].getAttribute('rtc-config');
-			const auRtcAttribute = ampAdElement[1].getAttribute('rtc-config');
-			const rowRtcAttribute = ampAdElement[2].getAttribute('rtc-config');
+			const usRtcAttribute: ObjectType = JSON.parse(
```

Comment:
defining what is parsed as `ObjectType` means we can try access `.urls` from that object. You will notice this is a common work around in the PR for anything that had implicitly `any`